lib/hca/enrollment_eligibility/status_matcher.rb

```diff
@@ -131,7 +133,10 @@ def process_text_match(text_matches, ineligibility_reason)
         nil
       end
 
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
       def parse(enrollment_status, ineligibility_reason = '')
+        return NONE if enrollment_status.blank?
         enrollment_status = enrollment_status.downcase.strip
 
         category_matcher = lambda do |statuses, enroll_status|
```

Comment:
minor but if you wanted to avoid the rubocop complexities you could extract the lambda out into another method:
```ruby
def category_matcher 
  lambda do |statuses, enroll_status|
    return statuses.include?(enroll_status) if statuses.is_a?(Array)
    return enroll_status == statuses
  end
end
```